src/arche/tools/helpers.py

```diff
@@ -36,7 +36,7 @@ def is_job_key(value):
 def ms_to_time(ms):
     if ms is None:
         return None
-    return str(d.timedelta(milliseconds=ms))
+    return str(d.timedelta(milliseconds=ms))[:-7] or str(d.timedelta(milliseconds=ms))
```

Comment:
`:-7` looks really magic here

Reply:
So this looked like the most elegant way to deal with 0 ms :)
I just want to remove those ms at all
```
str(d.timedelta(milliseconds=1000))

'0:00:01'

str(d.timedelta(milliseconds=1111))

'0:00:01.111000'

str(d.timedelta(milliseconds=12345678901))

'142 days, 21:21:18.901000'
```

Reply:
So, maybe placing it in a function stating the behavior.
Otherwise it looks really magic :stuck_out_tongue:

Reply:
The case when it doesn't work is only when the last 3 digits is 000 and I think it'll never be the case. So I removed or.